frontend_server/lib/server.dart

```diff
@@ -85,11 +85,6 @@ Future<int> starter(
       StringSink output,
     }) async {
   ArgResults options;
-  frontend.argParser
```

Comment:
Any reason to keep other references to it around?  e.g. https://github.com/flutter/engine/blob/01b8c078cb00f1b853c01a95405ca125b2114b1e/frontend_server/lib/server.dart#L124

Reply:
There's also quite a few references to this flag in the framework.

```shell
$ git grep -l track-widget-creation
dev/bots/test.dart
packages/flutter/lib/src/widgets/widget_inspector.dart
packages/flutter/test/widgets/widget_inspector_test.dart
packages/flutter_tools/bin/macos_build_flutter_assets.sh
packages/flutter_tools/bin/tool_backend.dart
packages/flutter_tools/bin/xcode_backend.sh
packages/flutter_tools/gradle/flutter.gradle
packages/flutter_tools/lib/src/android/gradle.dart
packages/flutter_tools/lib/src/commands/attach.dart
packages/flutter_tools/lib/src/commands/build_aot.dart
packages/flutter_tools/lib/src/commands/build_apk.dart
packages/flutter_tools/lib/src/commands/build_appbundle.dart
packages/flutter_tools/lib/src/commands/build_bundle.dart
packages/flutter_tools/lib/src/commands/run.dart
packages/flutter_tools/lib/src/commands/test.dart
packages/flutter_tools/lib/src/compile.dart
packages/flutter_tools/lib/src/runner/flutter_command.dart
```